### PR TITLE
Update docs for using Koalas entitysets

### LINKED ISSUE #1133: Document Koalas functionality

### PR DESCRIPTION
**Pull request by frances-h**
Update the docs for using Koalas dataframes for creating entitysets.
- Update FAQ regarding using Dask/Koalas
- Update performance doc to include Koalas entitysets as an option
- Add guide for creating Koalas entityset

Resolves #1133

### CODE REVIEW

File: docs/source/guides/using_koalas_entitysets.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+The ``Entity.add_interesting_values()`` method is not supported when using a Koalas ``Entity``.  If needed, users can manually set ``interesing_values`` on entities by assigning them directly with syntax similar to this: ``es["entity_name"]["variable_name"].interesting_values = ["Value 1", "Value 2"]``.
+
+EntitySet Limitations
+*********************
+When creating a Featuretools ``EntitySet`` that will be made of Koalas entities, all of the entities used to create the ``EntitySet`` must be of the same type, either all Koalas entities, all Dask entities, or all pandas entities. Featuretools does not support creating an ``EntitySet`` containing a mix of Koalas, Dask, and pandas entities.
+
+Additionally, the ``EntitySet.add_interesting_values()`` method is not supported when using a Koalas ``EntitySet``. Users can manually set ``interesing_values`` on entities, as described above.
```
Comment (thehomebrewnerd) [inline docs/source/guides/using_koalas_entitysets.rst:95]: `interesing_values` -> `interesting_values`

Comment (thehomebrewnerd) [reply docs/source/guides/using_koalas_entitysets.rst:95]: This also needs to be fixed in the Dask guide.

File: .circleci/config.yml [hunk 2]
```diff
@@ -210,8 +210,11 @@ jobs:
       name: python
       image_tag: << parameters.image_tag >>
     steps:
-      - run: sudo apt update && sudo apt install -y pandoc
+      - run: sudo apt update && sudo apt install -y pandoc
```
Comment (rwedge) [inline .circleci/config.yml:213]: extra whitespace at end of line

File: docs/source/guides/using_koalas_entitysets.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+This guide will provide an overview of how to create a Koalas ``EntitySet`` and then generate a feature matrix from it. If you are already familiar with creating a feature matrix starting from pandas dataframes, this process will seem quite familiar, as there are no differences in the process. There are, however, some limitations when using Koalas dataframes, and those limitations are reviewed in more detail below.
+
+Creating Entities and EntitySets
+--------------------------------
+Koalas ``EntitySets`` require Koalas and PySpark. Both can be installed directly with ``pip install featuretools[koalas]``. We will create a very small Koalas dataframe for this example. Koalas dataframes can also be created from pandas dataframes, or read in directly from a file.
```
Comment (rwedge) [inline docs/source/guides/using_koalas_entitysets.rst:14]: Should we also mention java / spark requirement?

**Review by rwedge — APPROVED** [review]
Looks good

**Pull request merged**